src/components/[guild]/EditGuild/EditGuildDrawer.tsx

```diff
@@ -106,12 +100,19 @@ const EditGuildDrawer = ({
     contacts,
     socialLinks,
     featureFlags: isSuperAdmin ? featureFlags : undefined,
+    tags: savedTags,
   }
   const methods = useForm<GuildFormType>({
     mode: "all",
     defaultValues,
   })
 
+  const { onSubmit: onTagsSubmit } = useEditTags({
```

Comment:
We could improve the `useEditTags` hook a bit I think: you can easily retrieve `savedTags` and `id` (guild ID) inside the `useEditTags` hook (with the `useGuild` hook), so you don't need to pass these values as props, and then this hook could accept only 1 param (`currentTags`).

Also, you could just watch the `tags` like this:

```ts
const tags = useWatch({ control: methods.control, name: "tags" });
```

And then pass the watched value to the `useEditTags` hook:

```ts
const { onSubmit: onTagsSubmit } = useEditTags(tags);
```